src/azul/deployment.py

```diff
@@ -564,8 +564,11 @@ def _log_client_response(self,
         return None
 
     @property
-    def _log_body_verbatim(self):
-        return boto3_log.isEnabledFor(logging.DEBUG)
+    def _log_body_debug_lvl(self) -> int:
```

Comment:
PL

Reply:
Method was drop in favor of simpler `config.debug` call, which achieves the desired logging effect at the desired debug level.